scripts/run.sh

```diff
@@ -13,10 +13,9 @@ fi
 
 set -x
 
-docker run \
--v $RWD_DATA:/opt/rwd-data \
--p 5000:5000 \
+docker-compose run \
 --rm \
-$ARGS \
-docker-rwd \
---reload
+-e RWD_DATA=$RWD_DATA \
+--name docker-rwd \
```

Comment:
Not sure where that came from. The container name doesn't matter much unless you need to use `exec` (which we don't).